static/js/components/tiles/bar_tile.tsx

```diff
@@ -141,9 +142,33 @@ export function BarTile(props: BarTilePropType): JSX.Element {
       replacementStrings={getReplacementStrings(barChartData)}
       className={`${props.className} bar-chart`}
       allowEmbed={true}
-      getDataCsv={
-        barChartData ? () => dataGroupsToCsv(barChartData.dataGroup) : null
-      }
+      getDataCsv={() => {
+        // Assume all variables will have the same date
+        const date =
+          props.variables.length > 0 ? props.variables[0].date : undefined;
+        const denoms = props.variables.map((v) => (v.denom ? v.statVar : ""));
```

Comment:
maybe also want to remove empty strings?

Reply:
Updated logic